USA/CA/beverly_hills/beverly_scraper.py

```diff
@@ -1,59 +1,13 @@
-import requests
-import os
-from bs4 import BeautifulSoup
-import urllib
-import re
-import time
 import sys
+import os
+import configs
 from pathlib import Path
 
 p = Path(__file__).resolve().parents[3]
-sys.path.insert(1, str(p) + '/common')
-from bs_scrapers.get_files import get_files
-
-__noted__ = 'fixes shamelessly stolen from dunnousername without credit' # Just don't delete this
-webpage = "http://www.beverlyhills.org/departments/policedepartment/crimeinformation/crimestatistics/web.jsp"
-'''
-Click the links that lead to the files, and copy their paths. **NOTE:** Ensure that files all match paths, otherwise remove a level until they match
-Also ensure that domain stays the same
-Verify on page that the href to the file contains the domain, if it doesn't, uncomment domain
-'''
-web_path = "/cbhfiles/storage/files/"
-domain = "http://www.beverlyhills.org"
-sleep_time = 5   # Set to desired sleep time
+sys.path.insert(1, str(p) + "/common")
+from list_pdf_scrapers.list_pdf_v2 import list_pdf_v2
 
 cur_dir = os.getcwd()
```

Comment:
You can should be able to replace these lines with `save_dir = "./data/"`

Reply:
Updated